life/gender/documentation-sexism.md

```diff
@@ -0,0 +1,22 @@
+# Documentation can be sexist
+
+This is one of those things that never occurred to me, but once someone pointed
+it out I realized I'm not that surprised. (Thanks @raq929)
+
+Part of it's that I'm new to tech, so a lot of the examples I've seen were
```

Comment:
Actually, that's all found so I'm just going to edit after merging. I don't think it's worth adding the ember docs link unless you want to, I mostly wanted you to just see it.